src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -1280,18 +1275,18 @@ def test_WINNF_FT_S_FDB_8(self, config_filename):
       scheduled_cpas_end_time = scheduled_cpas_start_time + timedelta(days=1,
                                                                       hours=CPAS_END_TIME -CPAS_START_TIME)
 
-    # Completion time in seconds
-    completion_time_in_secs = (scheduled_cpas_end_time - scheduled_cpas_start_time).seconds
+    # Run time in seconds
+    run_time_in_secs = (scheduled_cpas_end_time - scheduled_cpas_start_time).seconds
 
     # Wait until CPAS is scheduled to start
     logging.debug('Wait time for scheduled CPAS in (HH:MM:SS) %s',
```

Comment:
Small thing: "in" should be "is" (same on line 1287).

Reply:
Updated as per the comment.